externs/shaka/manifest_parser.js

```diff
@@ -93,6 +93,14 @@ shaka.extern.ManifestParser = class {
    * @exportDoc
    */
   onExpirationUpdated(sessionId, expiration) {}
+
+  /**
+   * Adds the ability to launch events to the parser.
+   *
+   * @param {function(!Event)} onEvent
+   * @exportDoc
+   */
+  setOnEvent(onEvent) {}
```

Comment:
I don't think this is needed.  There is already a callback on PlayerInterface called onEvent, which lets a ManifestParser dispatch events of its own to the application.

Reply:
I changed it!